style.css

```diff
@@ -3,6 +3,12 @@
 
 /* mobile header section */
 
```

Comment:
- [OPTIONAL] Your menu works great, :+1: :+1: I'll suggest you add a smooth scroll to your portfolio, it will make scrolling to different sections of the page when a user clicks on a menu item smoothly. You can achieve this by adding this line of CSS at the top of your CSS file. `html {
  scroll-behavior: smooth;
}`. :+1: :smile:

Reply:
thanks for the recommendation 👍 @V-Blaze